test/assertion.js

```diff
@@ -347,6 +347,25 @@ describe('Assertion', function(){
     });
   });
 
+  describe('.error(status, fn)', function(){
```

Comment:
Can you add a test case where both string and status are present if that's the intended functionality? (The documentation indicates it can accept all three.)